js/app.js

```diff
@@ -0,0 +1,36 @@
+"use strict";
+
+(function(){
+  angular
+  .module("radioBip", [
+    "ui.router",
+    "songs"
+  ])
+
+  .config([
+    "$stateProvider",
+    RouterFunction
+  ])
+
+  function RouterFunction($stateProvider){
+    $stateProvider
+    .state("radioBipIndex", {
+      url: "/songs",
+      templateUrl: "js/index.html",
+      controller: "IndexController",
```

Comment:
Doesn't matter too much in this case, but it'd be a good idea to include "Song" in the name of your controller. What would happen if you had another model (e.g., playlist, user) that also had an index controller?